main.py

```diff
@@ -1,10 +1,49 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    # We read the data from the players.json file
```

Comment:
There is no need to add these comments, code is self-explicit enough. Remove comments from this file, please